### DISCUSSION TITLE
Is deprecationWarningIssues still being used, do we want separate issues for deprecated dependencies?

### DISCUSSION THREAD

**Discussion by dword-design** [renovatebot/renovate#29737] [Request Help]
### Please tell us more about your question or problem

In the [doc](https://docs.renovatebot.com/configuration-options/#suppressnotifications) it says that it's possible to suppress dependency warning issues by setting `"suppressNotifications": ["deprecationWarningIssues"]`, which is also the default value. However looks like this isn't working and since deprecated dependencies are now shown in the dependency dashboard, this setting should either work or be removed from the doc to avoid confusion.

In general I like the idea to have a separate issue for deprecated dependencies or maybe for each deprecated one, which would be consistent with the rest of the tool where each dependency upgrade has its own pull request.

**Comment by rarkins** [comment]
The old functionality of one issue per deprecated dependency has been effectively disabled for years. I created this issue to remove it from code and docs: https://github.com/renovatebot/renovate/issues/29739

We may later add a new configuration option which could allow for:
- No deprecated dependency warnings at all
- Warnings in Dashboard (current functionality)
- Warnings in separate, combined issue
- One issue per deprecated package

e.g. a new option "deprecationWarnings" with enum values:
- "disabled"
- "dashboard"
- "combined-issue"
- "per-package"

**Reply by rarkins** [reply]
https://github.com/renovatebot/renovate/issues/29740

**Comment by dword-design** [comment]
This sounds cool
